app/models/event.rb

```diff
@@ -16,6 +16,7 @@ class Event < ActiveRecord::Base
   to_param :title
 
   validates :starts_at, presence: true
+  validates :location_id, presence: true
```

Comment:
@nicholasjhenry Sure thing. I ended up watching the whole talk. Funny one :)

As for the requested change, almost a year ago @mcgain has created a `location` method in the `Event` class, in effect overriding its `location` relationship like this:

```ruby
def location
  super || TBALocation.new
end
```

And it's giving me problems since that `TBALocation` object (which stands for "to-be-announced location) is not an `ActiveRecord` object and it explodes when attempting to validate the `location` relationship (`undefined method 'marked_for_destruction?' for #<TBALocation:0x007fa6eb88b050>`). 

Unless someone gives me a good reason to keep that `TBALocation` class, I'd like to remove it entirely.

On top of that, I would like to [renegotiate the requirements](https://vimeo.com/4552747): do we really need to make the location _required_ for an event? What if you want to create (i.e. persist it in db) an event today, but you still don't know where it's going to take place (the concept of a "to-be-announced" location). For that reason, I think there should *not* be a presence validation on the `location` (and naturally neither on `location_id`). What do you think?